db/migrate/20230430025441_add_archive_to_families.rb

```diff
@@ -0,0 +1,10 @@
+class AddArchiveToFamilies < ActiveRecord::Migration[7.0]
+  def up
+    add_column :families, :archived, :boolean
```

Comment:
This should be null: false and default: false. I think it's safe to ignore the strong\_migrations alarm here since the table is tiny.